### PR TITLE
FIXED: Bulk Delete page not showing full information

### PR DESCRIPTION
**Pull request by akemidx**
# Description
The Bulk Delete page had missing information:
1) showed user id instead of the person's name
2) showed incomplete information for a user's asset in they did not have a location set

Before:
Screenshot 2024-10-30 at 7 29 51 PM

After:
Screenshot 2024-10-30 at 7 29 29 PM

The location will be listed to match the location of the asset as it appears on the Assets index page.

Fixes # [26944](https://app.shortcut.com/grokability/story/26944/bulk-delete-assets-does-not-show-assigned-to)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
manual tests and test suite 

* PHP version: 8.1
* OS version: Sonoma 14.6

# Checklist:

- [x] I have read the Contributing documentation available here: https://snipe-it.readme.io/docs/contributing-overview
- [x] I have formatted this PR according to the project guidelines: https://snipe-it.readme.io/docs/contributing-overview#pull-request-guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: resources/views/hardware/bulk-delete.blade.php [hunk 1]
```diff
@@ -42,15 +42,15 @@
                 <td>{{ $asset->id }}</td>
                 <td>{{ $asset->present()->name() }}</td>
                 <td>
-                  @if ($asset->location)
+                  @if ($asset->location_id)
```
Comment (snipe) [inline resources/views/hardware/bulk-delete.blade.php]: Why was this line changed? There is a relationship (`$asset->location`) which is being checked for, which is generally better than just checking for the ID, since the ID could be invalid.

Comment (akemidx) [reply resources/views/hardware/bulk-delete.blade.php]: so, the two things we were checking against are the location_id and the rtd_location_id. these are the two values in the assets table when we looked for location. currently, we are ending up with blanks for the location of an asset on the bulk delete page, this is weird behavior, since it won't match the assets index page. since through `deployedLocationFormatter` we return the `row.name` off the `row.id`, and if not, then we pull the `rtd_location.name` from the `rtd_location.id`. its this if/else that we tried to parity.

Comment (snipe) [reply resources/views/hardware/bulk-delete.blade.php]: Right, but if the `location_id` or `rtd_location_id` reference a non-existent location (bad data, etc), that will 500. We should be checking against the relations to make sure those locations actually exist in the database.

Comment (akemidx) [reply resources/views/hardware/bulk-delete.blade.php]: righto, i'll get to this on monday and get that switched over. ty for the explaination.

**Comment by snipe** [comment]
Looks like we've got some tests failing here - can you take a look?
